docs/kyma/04-02-local-installation.md

```diff
@@ -19,88 +20,79 @@ Virtualization:
-
-7. Start the installation. Trigger the `run.sh` script to start Minikube with a Kyma-specific configuration and install the necessary components. Define the password used to log in to the Console UI using the `--password` flag. Run:
-    <div tabs>
-      <details>
-      <summary>
-      MacOS
-      </summary>
-
-      ```
-      ./cmd/run.sh --password {USER_PASSWORD}
-      ```
-      </details>
-      <details>
-      <summary>
-      Linux
-      </summary>
-
-      ```
-      ./cmd/run.sh --password {USER_PASSWORD} --vm-driver virtualbox
-      ```
-      </details>
-    </div>
-
-8. By default, the Kyma installation is a background process, which allows you to perform other tasks in the terminal window. Nevertheless, you can track the progress of the installation by running the `is-installed.sh` script. It is designed to give you clear information about the Kyma installation. Run it at any point to get the current installation status, or to find out whether the installation is successful.
-
-  ```
-  ./scripts/is-installed.sh
-  ```
-  >**TIP:** If the script indicates that the installation failed, try to install Kyma again by re-running the `run.sh` script. If the installation fails in a reproducible manner, don't hesitate to create a [GitHub](https://github.com/kyma-project/kyma/issues) issue in the project or reach out to the ["installation" Slack channel](https://kyma-community.slack.com/messages/CD2HJ0E78) to get direct support from the community.
-
-9. After the installation is completed, you can access the Console UI. Go to [this](https://console.kyma.local) address and select **Login with Email**. Use the **admin@kyma.cx** email address and the password you set using the `--password` flag.
-
-10. At this point, Kyma is ready for you to explore. See what you can achieve using the Console UI or check out one of the [available examples](https://github.com/kyma-project/examples).
+Follow these instructions to install Kyma from a release or from sources:
+<div tabs>
+  <details>
+  <summary>
+  From a release
+  </summary>
+    
+  1. Provision a Kubernetes cluster on Minikube. Run:
+    
+     ```bash
+     kyma provision minikube
+     ```
+     > **NOTE:** The `provision` command uses default Minikube VM driver installed for your operating system. For a list of supported VM drivers see [this document](https://kubernetes.io/docs/setup/minikube/#quickstart).
+  
+  2. Install Kyma on Minikube:
+     ```bash
+     kyma install 
+     ```
+ >**NOTE** If you want to install a specific release version, go to the [GitHub releases page](https://github.com/kyma-project/kyma/releases) to find out more about available releases. Use the release version as a parameter when calling ` kyma install --release {KYMA_RELEASE}`.
```

Comment:
This note doesn't really make sense, the `kyma install --release {KYMA_RELEASE}` should be the default. This flow is called "install from a release" after all.